ex1.c

```diff
@@ -42,13 +43,32 @@ int pcoord[2];
 /// the alpha coefficient used in the computation
 double alpha;
 
+double L=1.0;
+double source1[4]={0.4, 0.4, 0.2, 100};
```

Comment:
Not related to PDI, but for better understanding of the code, I would make a `struct` of heatSource for example.

```
struct heatSource {
    double heatCenterX;   // X-coordinate of the heat source center
    double heatCenterY;   // Y-coordinate of the heat source center
    double heatRadius;    // Radius of the heat source
    int heatStartIter;    // Iteration at which the heat source starts

    heatSource(double x, double y, double radius, int startIter)
        : heatCenterX(x), heatCenterY(y), heatRadius(radius), heatStartIter(startIter) {}
};
```

Reply:
If we want to do that, I think it is better to create an issue when all the tutorial are merged.